src/darwin/Framework/CHIPTests/MTROTAProviderTests.m

```diff
@@ -735,7 +757,7 @@ - (void)test004_DoBDXTransferDenyUpdateRequest
         XCTAssertEqualObjects(nodeID, @(kDeviceId1));
         XCTAssertEqual(controller, sController);
         XCTAssertEqualObjects(params.updateToken, updateToken);
-        XCTAssertEqualObjects(params.newVersion, @(18)); // TODO: Factor this out better!
+        XCTAssertEqualObjects(params.newVersion, kUpdatedSoftwareVersion); // TODO: Factor this out better!
 
         XCTAssertTrue([[NSFileManager defaultManager] contentsEqualAtPath:rawImagePath
                                                                   andPath:@"/tmp/chip-ota-requestor-downloaded-image1"]);
```

Comment:
We have a constant for this "where it will download the image to" path now, right?

Reply:
done